[BUG]: URL #include doesn't work in IDE mode

### Describe the bug

for files other than `example.cpp`:

```
/app/to.hpp:6:10: fatal error: raw.githubusercontent.com/cor3ntin/rangesnext/master/include/cor3ntin/rangesnext/to.hpp: No such file or directory
    6 | #include "raw.githubusercontent.com/cor3ntin/rangesnext/master/include/cor3ntin/rangesnext/to.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```



### Steps to reproduce

https://godbolt.org/z/hhfWfh9nq

### Expected behavior

Compilation succeeds

### Reproduction link

see above

### Screenshots

Not applicable

### Operating System

_No response_

### Browser version

_No response_

--- a/static/panes/executor.ts
+++ b/static/panes/executor.ts
@@ -373,11 +373,11 @@ export class Executor extends Pane<ExecutorState> {
                 }),
             );
         }
-        request.files.push(...moreFiles);
 
         Promise.all(fetches).then(() => {
             const treeState = tree.currentState();
             const cmakeProject = tree.multifileService.isACMakeProject();
+            request.files.push(...moreFiles);
 
             if (bypassCache) request.bypassCache = bypassCache;
             if (!this.compiler) {

